examples/acados/pendulum.py

```diff
@@ -111,7 +112,8 @@ def main():
     ocp = prepare_ocp(biorbd_model_path="models/pendulum.bioMod", final_time=1, n_shooting=100)
 
     # --- Solve the program --- #
-    sol = ocp.solve(solver=Solver.ACADOS)
+    opts = SolverOptionsACADOS()
```

Comment:
Is this to show what can one do? Because since this sends all default values, it is currently kind of useless?